### PR TITLE
Update for WV-224, add button ADMIN APP to Extension

### PR DESCRIPTION
**Pull request by stilwellmg**
Add a button labeled 'ADMIN APP' to the Extension and a listener that will open up the Voter Guide Possibility page from the Admin site for the Organization of the source page.

### CODE REVIEW

File: js/contentForeground/contentWeVoteUI.js [hunk 1]
```diff
@@ -1469,6 +1482,14 @@ function orgChoiceDialog (orgList) {
   });
 }
 
+async function openAdminPage () {
+  const { chrome: { runtime: { sendMessage, lastError } } } = window;
+  debugFgLog('ENTERING openAdminPage');
+  const state = await getGlobalState();
+  const { organizationWeVoteId, voterGuidePossibilityId, voterWeVoteId } = state;
+  window.open('https://api.wevoteusa.org/vg/create/?voter_guide_possibility_id=8852');
```
Comment (DaleMcGrew) [inline js/contentForeground/contentWeVoteUI.js]: Hi @stilwellmg you still have a hardcoded variable I think you want to replace with `voterGuidePossibilityId`

File: js/contentForeground/contentWeVoteUI.js [hunk 2]
```diff
@@ -318,22 +318,35 @@ function topMenu () {
...
     '  <div id="dlgAnchor"></div>' +
   '</div>';
   $('#topMenu').append(topMarkup);
 
   setSignInOutMarkup ();
+
+  // WV-224: Add event listener to the "Open Admin" button
+  $('#openAdminButton').click(async function () {
+    const state = await getGlobalState();
+    const { voterGuidePossibilityId} = state;
```
Comment (DaleMcGrew) [inline js/contentForeground/contentWeVoteUI.js]: Typically you would have a trailing space like this -- I would recommend fixing: 
const { voterGuidePossibilityId } = state;

Comment (stilwellmg) [reply js/contentForeground/contentWeVoteUI.js]: @DaleMcGrew - Thank you for catching these! The hardcoded url is left over from early, unsuccessful attempts that I thought I had removed. I will fix these items and resubmit the PR.

File: js/contentForeground/contentWeVoteUI.js (hunk 1 above)
Comment (stilwellmg) [reply js/contentForeground/contentWeVoteUI.js]: Fixed!

File: js/contentForeground/contentWeVoteUI.js (hunk 2 above)
Comment (stilwellmg) [reply js/contentForeground/contentWeVoteUI.js]: Fixed!

**Comment by DaleMcGrew** [comment]
👍 Thank you @stilwellmg!

**Pull request merged**